documentation/README.md

```diff
@@ -1523,6 +1525,26 @@ nth(1, [1, 2, 3]); // => 2
 </sup></div>
 
 
+### omit
+
+Creates a copy of the `object` without the given `props`.
+
+```js
+const omit = require('1-liners/omit');
+
+const object = {foo: 1, bar: 2, baz: 3};
+
+omit(['foo', 'baz'], object);  // => {bar: 2}
+object;                        // => {foo: 1, bar: 2, baz: 3}
```

Comment:
> object;                        // => {foo: 1, bar: 2, baz: 3}

nope, I meant the line `1538` is not necessary. Didn't realise that it was there before :blush: